Resfeber/CoreData/CoreDataStack.swift

```diff
@@ -9,7 +9,7 @@
 import Foundation
 import CoreData
 
-class CoreDataStack {
+open class CoreDataStack {
```

Comment:
Why are you adding `open` here?

Reply:
I added the `open` access level to resolve an error that was occuring when attempting to subclass `CoreDataStack` in the unit tests target.

"Marking a class as open explicitly indicates that you’ve considered the impact of code from other modules using that class as a superclass, and that you’ve designed your class’s code accordingly."
Excerpt from [swift.org](https://docs.swift.org/swift-book/LanguageGuide/AccessControl.html)

Reply:
I just took another look at the unit tests and noticed the `TestCoreDataStack.swift` file does not include `@testable` when importing the `Resfeber` target. I will update the file with `@testable` and remove the `open` access level modifier here.

Reply:
```suggestion
class CoreDataStack {
```

Reply:
```suggestion
class CoreDataStack {
```